propnet/core/symbols.py

```diff
@@ -198,3 +199,20 @@ def provenance(self):
             (id): time of creation of the Symbol
         """
         return self._provenance
+
+    def __hash__(self):
+        return hash(self.type.name) + hash(self.value)
+
+    def __eq__(self, other):
+        tol = 1E-6
```

Comment:
This can be simplified, something like:

```
def __eq__(self, other):
    return (self.type == other.type) and np.isclose(self.value, other.value)
```

Of course, that won't work for testing objects for equality, so perhaps some logic is needed for that.

Reply:
I've made the changes requested above.
I'm confused about what won't work for testing equality? 
I added this equals method because I needed to use it for testing purposes.